src/effects/effectslot.cpp

```diff
@@ -179,8 +181,6 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
             pParameter->loadEffect(pEffect);
         }
 
-        slotEffectMetaParameter(m_pControlMetaParameter->get(), true);
```

Comment:
> If we remove the "true" this line does nothing and can be removed.

It will do something [if the effect is disabled](https://github.com/daschuer/mixxx/blob/5fdb8284018b7f71f38e83ba2f55149a4809ae6f/src/effects/effectslot.cpp#L264).